packages/components/pie-thumbnail/test/component/pie-thumbnail.spec.ts

```diff
@@ -1,7 +1,23 @@
 import { test, expect } from '@playwright/test';
 import { BasePage } from '@justeattakeaway/pie-webc-testing/src/helpers/page-object/base-page.ts';
+import { getShadowElementStylePropValues } from '@justeattakeaway/pie-webc-testing';
+import { type ThumbnailProps } from '../../src/index.ts';
```

Comment:
I think this is causing your test issues for some reason. Can you try and import this directly from `../../src/defs.ts`